plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -904,7 +905,8 @@ public void registerCoreMembers() {
         //
         // @Description
         // This command creates a new minecraft world with the specified name, or loads an existing world by thet name.
-        // TODO: Document Command Details (generator)
+        // Specify a generator to change the terrain.
+        // If a generator is not specified, it defaults to the normal generated world.

```

Comment:
Documentation here should explain what a generator is, what you can specify as input...